src/App.js

```diff
@@ -37,14 +51,20 @@ function App() {
   };
 
   return (
-    <>
-      <h1>Todo List</h1>
-      <h3>
-        Date: {today.getMonth()}/{today.getDate()}/{today.getFullYear()}
-      </h3>
-      <AddTodoForm onAddTodo={addTodo} />
-      <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
-    </>
+    <div className="todo-wrapper">
+      {isLoading ? (
+        <p>Loading...</p>
+      ) : (
+        <div>
+          <h1>Todo List</h1>
+          <h3>
+            Date: {today.getMonth() + 1}/{today.getDate()}/{today.getFullYear()}
+          </h3>
+          <AddTodoForm onAddTodo={addTodo} />
+          <TodoList todoList={todoList} onRemoveTodo={removeTodo} />
+        </div>
+      )}
+    </div>
```

Comment:
You should condition if either "Loading..." or TodoList component is rendered:

_Using a ternary operator inside JSX, if isLoading is true render the loading message, otherwise render the **TodoList component**_

You could also put something like: "Loading your todos..."